sql/check_duplicated_foreign_keys.sql

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2019-2024. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health-sql
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+with
+-- combine fk constraints with the attributes involved in them
+fk_with_attributes as (
+        select
+            c.conname as fk_name,
+            c.conrelid,
+            c.confrelid,
+            fk_conkey.conkey_order as att_order,
+            fk_conkey.conkey_number,
+            fk_confkey.confkey_number,
+            rel_att.attname as rel_att_name,
+            rel_att.atttypid as rel_att_type_id,
+            rel_att.atttypmod as rel_att_type_mod,
+            rel_att.attnotnull as rel_att_notnull,
+            frel_att.attname as frel_att_name,
+            frel_att.atttypid as frel_att_type_id,
+            frel_att.atttypmod as rel_att_type_mod,
+            frel_att.attnotnull as rel_att_notnull
+        from pg_catalog.pg_constraint as c
+            cross join lateral unnest(c.conkey) with ordinality as fk_conkey(conkey_number, conkey_order)
```

Comment:
Try to ignore parse error `-- noqa: disable=PRS`

See also https://github.com/mfvanek/pg-index-health-sql/blob/6a31f62303b2c7d7416fdf98a6fe4820adda4a8e/sql/bloated_indexes.sql#L20

Reply:
I added the following, but it doesn't seem to resolve the lint.

Reply:
Parse error has gone.
There are still some linter errors
```
  == [/github/workspace/sql/check_duplicated_foreign_keys.sql] FAIL
  L:  27 | P:  14 | ST09 | Joins should list the table referenced later first.
                         | [structure.join_condition_order]
  L:  27 | P:  39 | AL01 | Implicit/explicit aliasing of table. [aliasing.table]
  L:  28 | P:  65 | AL01 | Implicit/explicit aliasing of table. [aliasing.table]
  L:  28 | P:  77 | LT01 | WARNING: Expected single whitespace between naked
                         | identifier and start bracket '('. [layout.spacing]
  L:  29 | P:  65 | AL01 | Implicit/explicit aliasing of table. [aliasing.table]
  L:  29 | P:  78 | LT01 | WARNING: Expected single whitespace between naked
                         | identifier and start bracket '('. [layout.spacing]
  L:  31 | P:  47 | AL01 | Implicit/explicit aliasing of table. [aliasing.table]
  L:  33 | P:  17 | LT03 | Found leading binary operator. Expected only trailing
                         | near line breaks. [layout.operators]
  L:  34 | P:  47 | AL01 | Implicit/explicit aliasing of table. [aliasing.table]
  L:  36 | P:  17 | LT03 | Found leading binary operator. Expected only trailing
                         | near line breaks. [layout.operators]
  L:  40 | P:   5 | LT08 | Blank line expected but not found after CTE closing
                         | bracket. [layout.cte_newline]
  L:  50 | P:   1 | LT08 | Blank line expected but not found after CTE closing
                         | bracket. [layout.cte_newline]
  L:  54 | P:   6 | ST09 | Joins should list the table referenced later first.
                         | [structure.join_condition_order]
  L:  54 | P:  33 | AL01 | Implicit/explicit aliasing of table. [aliasing.table]
  L:  55 | P:  43 | AL01 | Implicit/explicit aliasing of table. [aliasing.table]
  L:  57 | P:   9 | LT03 | Found leading binary operator. Expected only trailing
                         | near line breaks. [layout.operators]
  L:  58 | P:   9 | LT03 | Found leading binary operator. Expected only trailing
                         | near line breaks. [layout.operators]
  L:  59 | P:   9 | LT03 | Found leading binary operator. Expected only trailing
                         | near line breaks. [layout.operators]
  L:  60 | P:  36 | AL01 | Implicit/explicit aliasing of table. [aliasing.table]
  All Finished!
```

Reply:
![image](https://github.com/user-attachments/assets/6792841f-61a7-404b-8b8e-05973786c529)

Are the ci of local and remote different? Lint error does not occur in local

Reply:
> Are the ci of local and remote different? Lint error does not occur in local

It can because in CI newer image used.